src/codegen/templates/_task_modules/out_stream.py.mako

```diff
@@ -165,3 +165,9 @@ ${property_template.script_property(attribute)}\
         return self._interpreter.write_raw(
             self._handle, number_of_samples_per_channel,
             self.auto_start, self.timeout, numpy_array)
+
+    def get_channels_buffer_size (self):
+        in_stream = InStream(self._task, self._interpreter) 
+        channels_to_read = in_stream.channels_to_read
```

Comment:
`channels_to_read` will not work for write properties. In AO and CO tasks it probably returns an error, and in DIO tasks it may have the wrong channels even if no error.